### ISSUE TITLE
Leading spaces not preserved in text blocks

### ISSUE THREAD

**Issue by MV10** [flux3dp/beam-studio#26]
I have a piece where a multi-line text block needs leading spaces to clear a vector image. When I save the layout to a file and reload it, those leading spaces are lost. For example:

```
A long line of text
Another long line of text
          Some text with leading spaces
          More text with leading spaces
No leading spaces
```

Saving and re-loading that results in everything being left-justified.

**Comment by net5566** [comment]
It's fixed on 1.6.1 beam studio.

**Comment by MV10** [comment]
@net5566 is there an ETA for that version? The Flux download page still shows 1.4.6.
